<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>start_panel</class>
 <widget class="QMainWindow" name="start_panel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>843</width>
    <height>570</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>829</width>
      <height>501</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>眼动仪</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QFrame" name="frame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>450</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>450</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QPushButton" name="btn_EyetrackStart">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>400</y>
            <width>113</width>
            <height>29</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>启动眼动仪</string>
          </property>
         </widget>
         <widget class="QPlainTextEdit" name="eye_tracker_info">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>50</y>
            <width>361</width>
            <height>271</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
         </widget>
        </widget>
        <widget class="QFrame" name="frame_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>450</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>450</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QPushButton" name="btn_Calibration">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>400</y>
            <width>92</width>
            <height>29</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>开始矫正</string>
          </property>
         </widget>
         <widget class="QProgressBar" name="distance_show">
          <property name="geometry">
           <rect>
            <x>370</x>
            <y>50</y>
            <width>21</width>
            <height>271</height>
           </rect>
          </property>
          <property name="minimum">
           <number>45</number>
          </property>
          <property name="maximum">
           <number>75</number>
          </property>
          <property name="value">
           <number>60</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="format">
           <string>%v</string>
          </property>
         </widget>
         <widget class="QLabel" name="distance_text">
          <property name="geometry">
           <rect>
            <x>370</x>
            <y>330</y>
            <width>21</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>主观实验</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>801</width>
        <height>181</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>-1</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="toolTipDuration">
       <number>-1</number>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {
    font: 20px &quot;黑体&quot;;
	background-color: rgb(240, 240, 240);
    border-radius:6px;
    margin-top:12px;
    border-width: 1px;
    border-style: solid;
    border-color: rgb(255, 0, 0);
}
QGroupBox:title {
    color:rgb(28, 120, 167);
    subcontrol-origin: margin;
    left: 10px;
}</string>
      </property>
      <property name="title">
       <string>数据库</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>151</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>数据库地址：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>70</y>
         <width>641</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btnDirget">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>120</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>设置地址</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnApply">
       <property name="geometry">
        <rect>
         <x>650</x>
         <y>120</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>应用</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="btnStart">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>410</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>开始实验</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPreStart">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>320</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>开始预热实验</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>843</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="action_frequency"/>
    <addaction name="action_filter"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_frequency">
   <property name="text">
    <string>设置频率</string>
   </property>
  </action>
  <action name="action_filter">
   <property name="text">
    <string>设置滤波器</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
